### PR TITLE
Flex e

### PR DESCRIPTION
**Pull request by asrarkharal**
can you see this idea.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,14 @@
-hello
```
Comment (asrarkharal) [inline index.html:25]: this should be removed

File: index.html [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
-lets see
```
Comment (asrarkharal) [inline index.html:26]: this line is not good.

**Comment by asrarkharal** [comment]
ok

**Comment by eyahya-khan** [comment]
thanks..

**Pull request merged**